test/tests/preferences_syncTests.js

```diff
@@ -0,0 +1,73 @@
+describe("Sync Preferences", function () {
+	describe("Settings", function () {
```

Comment:
It's inline with how other preference tests are scoped for different subsections. I.e. this is the *Settings* tab, not *Reset*.